dev-db/redict/redict-7.3.0.ebuild

```diff
@@ -0,0 +1,159 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# N.B.: It is no clue in porting to Lua eclasses, as upstream have deviated
+# too far from vanilla Lua, adding their own APIs like lua_enablereadonlytable
```

Comment:
My hope is that perhaps this will end up fixed in redict. Consider bringing it up?

Reply:
de-vendoring lua is def. on the todo for redict, i'll attempt to when i get some free time

it's quite a bit more work than hiredict/jemalloc so it'll take a bit tho